src/steps/initializeBranchProtectionSettings.test.ts

```diff
@@ -11,6 +11,7 @@ const createMockOctokit = (request: SpyInstance) =>
 const stubValues = { owner: "", repository: "" };
 
 describe("migrateBranchProtectionSettings", () => {
+	// eslint-disable-next-line vitest/expect-expect
```

Comment:
The lint rule is right though. Per https://vitest.dev/api/expect.html#rejects the right test API is:

```ts
await expect(
	initializeBranchProtectionSettings(
		createMockOctokit(mockRequest),
		stubValues,
	),
).resolves.not.toThrow();
```

Let's go with that?

Reply:
yeah, that makes sense. 
I can go ahead and update it 👍